external/jacoco/README.md

```diff
@@ -0,0 +1,22 @@
+Running External tests locally
+ To run any AQA tests locally, you follow the same pattern:
+
+ Ensure your test machine is set up with test prereqs. For external tests, you do need Docker installed.
```

Comment:
Please add a hyperlink for test prereqs to https://github.com/AdoptOpenJDK/openjdk-tests/blob/master/doc/Prerequisites.md  (you do this in .md files by surrounding the text you want the hyperlink on with square brackets [ ] followed directly by the URL in braces ( )

`[test prereqs](https://github.com/AdoptOpenJDK/openjdk-tests/blob/master/doc/Prerequisites.md)` then displays as 
[test prereqs](https://github.com/AdoptOpenJDK/openjdk-tests/blob/master/doc/Prerequisites.md)